packages/docs/src/page-configs/ui-elements/tree-view/examples/CustomizableContent.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <va-tree-view
+    :nodes="nodes"
+    node-key="id"
```

Comment:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/5394573/183219230-658a64ec-1650-4ef4-b466-c1b3cd5d089f.png">

Can't we use logic from select? The way I see it it's almost the same as for select.